src/components/progress/themes/circular/circular.progress.base.scss

```diff
@@ -61,30 +79,39 @@
 }
 
 [part~='fill'] {
+    --_progress-percentage: calc(var(--_progress-whole, 0) / 100);
+
     stroke-dasharray: #{$circumference} #{$circumference};
-    stroke-dashoffset: calc(#{$circumference} - var(--percentage) * #{$circumference});
-    transition: stroke-dashoffset var(--duration) linear;
-    stroke-width: calc(var(--stroke-thickness) + rem(0.75px));
+    stroke-dashoffset: calc(#{$circumference} - var(--_progress-whole, 0) * #{$circumference});
+    transition: stroke-dashoffset var(--_transition-duration) linear;
+    stroke-width: calc(#{var(--stroke-thickness)} + rem(0.75px));
+}
+
+[part='svg indeterminate'] {
+    transform-origin: 50% 50%;
+    animation: 3s linear 0s infinite $animation-direction none running rotate-center;
 
-    --percentage: 0;
-    --duration: 500ms;
+    [part~='fill'] {
+        stroke-dashoffset: calc(#{$circumference} * 2);
+        animation: indeterminate-accordion 1.5s cubic-bezier(0, 0.085, 0.68, 0.53) $animation-direction infinite;
+    }
 }
 
+:host-context([dir='rtl']),
```

Comment:
As far as I am aware, this does not yet work in Safari and Firefox?

Reply:
They don't support it and AFAIK they have no plans to ever do. Not to mention it does not correctly handles `auto` and inherited values.

[`:dir`](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) is the one which should be used here.

Reply:
@desig9stein This should be fixed.